varats/varats/projects/c_projects/libjpeg-turbo.py

```diff
@@ -0,0 +1,59 @@
+"""Project file for libjpeg-turbo."""
+import typing as tp
+
+import benchbuild as bb
+from benchbuild.utils.cmd import make, cmake
+from benchbuild.utils.settings import get_number_of_jobs
+from plumbum import local
+
+from varats.paper_mgmt.paper_config import project_filter_generator
+from varats.project.project_util import (
+    ProjectBinaryWrapper,
+    wrap_paths_to_binaries,
+    BinaryType,
+)
+from varats.provider.cve.cve_provider import CVEProviderHook
+from varats.utils.settings import bb_cfg
+
+
+class Libxml2(bb.Project, CVEProviderHook):  # type: ignore
+    """libjpeg-turbo is a JPEG image codec."""
+
+    NAME = 'libjpeg-turbo'
```

Comment:
```suggestion
    NAME = 'libjpeg_turbo'
```
A, one thing I missed. We also need to use a `_` here, otherwise, our NAME detection breaks^^
I probably should setup a test that no wrong project names are introduced :laughing: (se-passau/VaRA#721)